docs/inputs.md

```diff
@@ -176,16 +176,6 @@ To use an SVGO config file in a folder:
     svgo-config: path/to/svgo.config.js
 ```
 
-To use an SVGO config file in the YAML format (e.g. if you're using SVGO v1):
-
-```yml
-# .github/workflows/optimize.yml
-
-- uses: ericcornelissen/svgo-action@v3
-  with:
-    svgo-config: .svgo.yml
-```
```

Comment:
TODO: Does this also mean that YAML parsing is no longer needed by this project?

Reply:
Per the [v2.0.0](https://github.com/svg/svgo/releases/tag/v2.0.0) release notes, YAML-based configuration files are no longer supported by SVGO (and [v3.0.0](https://github.com/svg/svgo/releases/tag/v3.0.0) didn't change that), so there's indeed no reason for this action to keep support for YAML-based configuration files.

Reply:
This thread was resolved by 46a66bcaa0bee0f0a37d855e674bf6fedbe9eed0